### DISCUSSION TITLE
Deleting projects and fields

### DISCUSSION THREAD

**Discussion by IdkGoodName** [community/community#7680] [Projects and Issues]
So I was testing out new spreadsheet projects and found 2 things:
- <sub>~~You can't delete projects. While I understand this somewhat, it's a bit annoying when you have created a testing project and can't delete it. This also applies to other type of project.~~</sub> ***(Apparently this has been already mentioned 3 times)***
- Fields cannot be deleted. I can't seem to find a way to delete spreadsheet fields.

**Comment by jthompson-kinship** [comment]
~Having same issue deleting fields, would be nice~
Looks like this works now, I had to reload my project to get the delete option, but now it's fine
